vars.production.yml

```diff
@@ -4,4 +4,5 @@ memory: 64M
 routes:
   - route: www-redirects-prod-datagov.app.cloud.gov
   - route: climate.data.gov
+  - route: dashboard.data.gov
   - route: www.data.gov
```

Comment:
> - route: www.data.gov

do we need to remove this line?

Reply:
I don't think so. I believe we want `www` traffic routed through this app still, right?

Reply:
I'm not sure. @GSA/data-gov-support thoughts?

Reply:
This is really a custom router for us, that we use for any domain/route that needs to be re-routed in the short/medium term. It doesn't matter whether it's catalog, dashboard, climate, etc.

Reply:
Sorry, you quoted `www` so I interpreted it as asking if we need the `www` route on the website app. If you mean `dashboard`, I think long term we would want to remove it, yes.

Reply:
Ha. No, I was uncertain if keeping `www.data.gov` in this app would interfere with catalog. So long as we're using it as James mentioned, I'm clear 👍🏼